app/models/profile_field.rb

```diff
@@ -22,7 +22,7 @@ class ProfileField < ApplicationRecord
   private
 
   def generate_attribute_name
-    self.attribute_name = label.titleize.scan(WORD_REGEX).join.underscore
+    self.attribute_name = "attribute_#{SecureRandom.uuid.delete('-')}"
```

Comment:
I'm going to guess that since the existing `attribute_name` is working, this new change doesn't need to be retroactively applied?

Reply:
correct, what's changing is (1) how we generate attribute names on creation, (2) the assumptions we have in tests and seeds about attribute names for a field given a label (no longer related).